### PR TITLE
small tweaks

### PR DESCRIPTION
**Pull request by LarkAscending**
1. Merge Specifying Fonts and Aligning Text.
1. Spell out assumptions, including bold24Roboto variable.
1. Change BoxShadow example to show multiple shadows.

### CODE REVIEW

File: web-analogs.md [hunk 1]
```diff
@@ -162,8 +162,11 @@ var container = new Container( // grey box
 ### Setting Container Width
 
 To specify the width of a [Container](https://docs.flutter.io/flutter/widgets/Container-class.html)
-widget you set its ```width``` property. This is equivalent to specifying the CSS 
-max-width property.
+widget you set its ```width``` property. This is a static width, unlike the
```
Comment (sethladd) [inline web-analogs.md]: you only need single back-ticks for inline code. triple backticks are useful for multi-line code.

Comment (sethladd) [inline web-analogs.md]: Is there another word, other than "static" ? That could be confused with the language's semantics for "static".

File: web-analogs.md [hunk 2]
```diff
@@ -9,16 +9,42 @@ permalink: /web-analogs/
 {:toc}
 
 <div class="begin-examples"></div>
-This page is for users who are familiar with the HTML and CSS syntax for arranging components of an application's UI. It maps HTML/CSS code snippets to their Flutter/Dart code equivalents.
+This page is for users who are familiar with the HTML and CSS syntax for 
+arranging components of an application's UI. It maps HTML/CSS code snippets to 
+their Flutter/Dart code equivalents.
+
+The examples assume:
+* The HTML doctype is set to HTML5, and the CSS box model for all HTML elements
```
Comment (Hixie) [inline web-analogs.md]: "HTML5" is an obsolete buzzword, I would go with just "The HTML document starts with a modern HTML DOCTYPE, ..."

File: web-analogs.md [hunk 3]
```diff
@@ -9,16 +9,42 @@ permalink: /web-analogs/
...
 
 <div class="begin-examples"></div>
-This page is for users who are familiar with the HTML and CSS syntax for arranging components of an application's UI. It maps HTML/CSS code snippets to their Flutter/Dart code equivalents.
+This page is for users who are familiar with the HTML and CSS syntax for 
+arranging components of an application's UI. It maps HTML/CSS code snippets to 
+their Flutter/Dart code equivalents.
+
+The examples assume:
+* The HTML doctype is set to HTML5, and the CSS box model for all HTML elements 
+is set to [quirks mode](https://css-tricks.com/box-sizing/), for consistency
```
Comment (Hixie) [inline web-analogs.md]: "border-box", not "quirks mode". "quirks mode" is a different thing entirely.

File: web-analogs.md (hunk 1 above)
Comment (Hixie) [inline web-analogs.md]: I'd go with "fixed"

File: web-analogs.md [hunk 4]
```diff
@@ -516,7 +550,7 @@ var container = new Container( // grey box
 The following examples show how to make and customize shapes.
 
 ### Rounding Corners
-To round the corners of a rectangular shape, use the borderRadius ```property``` of a
+To round the corners of a rectangular shape, use the ```borderRadius``` property of a
 [BoxDecoration](https://docs.flutter.io/flutter/painting/BoxDecoration-class.html) 
 widget. Create a new [BorderRadius](https://docs.flutter.io/flutter/painting/BorderRadius-class.html)
```
Comment (Hixie) [inline web-analogs.md]: BoxDecoration isn't a widget.

**Comment by Hixie** [comment]
LGTM

File: web-analogs.md (hunk 1 above)
Comment (LarkAscending) [inline web-analogs.md]: Agreed.

Comment (LarkAscending) [inline web-analogs.md]: Yes, will handle this in a separate PR already in the queue.

**Pull request merged**